Cubical/Categories/Instances/PosetCat/MonotoneAdjoint.agda

```diff
@@ -0,0 +1,105 @@
+{-# OPTIONS --safe #-}
+
+-- defines adjoint for monotone functions and morphisms in the Adjoint category
+
+module Cubical.Categories.Instances.PosetCat.MonotoneAdjoint where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.HLevels
+
+open import Cubical.Relation.Binary.Poset
+open import Cubical.Data.Sigma
+open import Cubical.HITs.PropositionalTruncation.Base
+open import Cubical.HITs.PropositionalTruncation.Properties
+
+open import Cubical.Categories.Instances.PosetCat.Monotone
+
+
+private
+  variable
+    ℓ ℓ' : Level
+
+
+module _ {ℓ ℓ' : Level} where
+
+  -- adjoints for monotone functions
+  record _⊣_ {X Y : Poset ℓ ℓ'} (L : MonFun X Y) (R : MonFun Y X) : Type (ℓ-max ℓ ℓ') where
+    field
+      adjIff : ∀ {x y} → Iso
+        ((PosetStr._≤_ (Y .snd)) (MonFun.f L x) y)
+        ((PosetStr._≤_ (X .snd)) x (MonFun.f R y))
```

Comment:
There's also a version of the universal element definition of adjoints for monotone functions and similar theorems to what yall did with representable profunctors. That stuff will probably be useful when we get to meets/joins/Heyting impl. We should think about if we can reuse the category stuff for that.